jQuery/domManipulation.js

```diff
@@ -0,0 +1,65 @@
+window.honey = {};
+
+window.honey.selectors = {
+  menu: '.menu',
+  content: '.content',
+  paginator: '.paginator',
+  pageSelect: '#page-select',
+  firstPageButton: '#first-page-button',
+  previousPageButton: '#previous-page-button',
+  nextPageButton: '#next-page-button',
+  lastPageButton: '#last-page-button',
+  nameInput: '#name-input',
+  characterSelect: '#character-select',
+  hometownSelect: '#hometown-select',
+  sizeSelect: '#size-select'
+};
+
+/* Defines functions that interact directly with the DOM */
+window.honey.domManipulation = window.honey.domManipulation || (function($, selectors) {
+  function populateSelectWithOptions(select, options) {
+    options.forEach(function(option) {
+      $(select).append($('<option>', { value: option, html: option }));
+    });
+  }
+
+  function gatherParams(form) {
+    var name      = form.find(selectors.nameInput);
+    var character = form.find(selectors.characterSelect);
+    var hometown  = form.find(selectors.hometownSelect);
+    var size      = form.find(selectors.sizeSelect);
+
+    return [
+      { field: 'name', value: name.val() },
+      { field: 'character', value: character.val() },
+      { field: 'hometown', value: hometown.val() },
+      { field: 'size', value: size.val() }
+    ];
+  }
+
+  function redrawContent(content, citizens) {
```

Comment:
both this function and `redrawPageSelect` can actually be made generic

```js

function redraw(element, afterward, data) {
  $(element).empty() // or element.innerHtml = ""; 

  // redraw and pass in custom data. 
  afterward(element, data); 
}
```

`redrawContent` becomes:

```js
  redraw($("some-element"), function(element, data) {
    var citizens = data.citizens;

    citizens.each(function(citizen) {
      element.append($('<div>', { class: "citizen", html: citizen.name }));
    })
  });
```

`redrawPageSelect` becomes

```js
redraw($("#page-select"), function(element, data) {
  var current  = data.page.current,
        max   = data.page.max;
  
  while (current <= max) {
    element.append($('<option>', { value: current, html: current }));
    current++;
  }
});
```

It's technically more verbose since you'll be replacing the `redrawPageSelect` function call, but it's more explicit exactly what happens! food for thought.